daemon/process/source/Main.cpp

```diff
@@ -253,8 +253,21 @@ static void parseArgs(int argc, char **argv)
  */
 static std::shared_ptr<Settings> createSettings()
 {
-    // create settings from either defaults or JSON settings file
+    // create settings from either defaults, development or JSON settings file
     std::shared_ptr<Settings> settings;
+
+#if (AI_BUILD_TYPE == AI_DEBUG)
```

Comment:
currently we can't rely on AI_BUILD_TYPE=AI_DEBUG to distinguish between PROD and VBN builds. We need a dedicated compile flag to allow "/opt/dobby.json"

Reply:
Added ENABLE_OPT_SETTINGS compilation flag.